packages/flutter/lib/src/widgets/container.dart

```diff
@@ -385,6 +386,10 @@ class Container extends StatelessWidget {
 
   /// The transformation matrix to apply before painting the container.
   final Matrix4 transform;
+  /// clipBehaviour for [BoxDecoration.borderRadius]
```

Comment:
Nit: blank line above this, and a full sentence description
```suggestion
  
  /// clipBehaviour for [BoxDecoration.borderRadius]
```